client/infinite_scroll.js

```diff
@@ -0,0 +1,25 @@
+const DEFAULT_DEBOUNCE = 200;
+
+TemplateController('infinite_scroll', {
+  onCreated() {
+    showMoreVisible = (eventName) => {
```

Comment:
Please always use declared variables instead of creating "globals" like this. In this case you can just assign it to `this.showMoreVisible` or better naming would be `this.checkIndicatorVisiblity`